test/compute-engine/simplify.test.ts

```diff
@@ -937,53 +785,94 @@ const RULE_TEST_CASES: TestCase[] = [
 const RULES_USED: string[] = [];
 
 const RULES: Rule[] = [
-  '(-x)^n:even -> x^n',
-  '(-x)^n:odd -> -(x^n)',
+  /*@fixme
```

Comment:
What is the problem with these rules? Are they not being run, or do they give an unexpected result, or...?

Reply:
They are not being run